index.html

```diff
@@ -4,6 +4,39 @@
     <title>The Best Recipes!</title>
   </head>
   <body>
-    <h1>Recipes</h1>
+    <h1>Recipes - Selected Japanese and Italian Cuisines</h1>
+    <br>
+    <div>
+      <h2>Japanese Cuisine Recipes</h2>
+    </div>
+    <br>
+    <div> 
```

Comment:
I think this div is not necessary. Ul is a block element already, just like Div. So the div does not serve a proper purpose here imo.